lightgbm/src/test/resources/log4j.properties

```diff
@@ -5,4 +5,6 @@ log4j.appender.stdout.layout.ConversionPattern=%d{HH:mm:ss} %-5p %c{1}:%L - %m%n
 
 log4j.rootLogger=WARN, stdout
 log4j.logger.org.apache.spark=WARN, stdout
+log4j.additivity.org.apache.spark=false
 log4j.logger.com.microsoft=INFO, stdout
+log4j.additivity.com.microsoft=false
```

Comment:
What do these lines and file do?

Reply:
Without these lines, every log entry gets printed twice in the output.  Without the file, there are no logs for tests at all.